resources.md

```diff
@@ -57,3 +57,42 @@ title: Ontology Tools and Resources
 - [Linked Data Patterns: A pattern catalogue for modelling, publishing, and consuming Linked Data](http://patterns.dataincubator.org/book/) by Leigh Dodds and Ian Davis
 - [Learning SPARQL](http://www.learningsparql.com/) by Bob DuCharme
 - [Guidelines for writing definitions in ontologies](https://philpapers.org/archive/SEPGFW.pdf) by Selja Seppälä, Alan Ruttenberg and Barry Smith
+
+## Related Projects
```

Comment:
You moved "legacy" information into the main resources section. Are you sure this is intended? Isnt it possible the information is a bit outdated? Like I don't know what GMOD is, but https://www.alliancegenome.org/ I do know (which seems to be the same, but newer)

Reply:
Unless you see a problem (i.e. it is actually somewhere on the life site), maybe just remove.

Reply:
i'm happy to delete this entirely, I agree this is a bit out of date.

Reply:
deleted in 347e73b, they're really not that relevant these days anyway